### DISCUSSION TITLE
I followed Security Onion Essentials - Intro to Analyst Tools

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#5977] [Unsupported Versions] [answered]
I heard that there where workflow #'s. I searched for these and see many references to workflows but I am looking for the ones mentioned in this youtube that take these tools and shows examples on how to use these tools? 
thanks

**Comment by dougburks** [comment, marked as answer]
Security Onion Essentials is an 8-video playlist:
https://www.youtube.com/watch?v=5fxVaVO8-L8&list=PLljFlTO9rB155aYBjHw2InKkSMLuhWpxH

Intro to Analyst Tools is the fourth video in the playlist:
https://www.youtube.com/watch?v=c18D7Dkoz5E&list=PLljFlTO9rB155aYBjHw2InKkSMLuhWpxH&index=4

In that video, when Josh mentions workflow numbers, he is referring to the next videos in the playlist.

From https://www.youtube.com/watch?v=OZ2PR4NiOn8&list=PLljFlTO9rB155aYBjHw2InKkSMLuhWpxH&index=5:
> In this session, we cover the first of three common workflows in Security Onion - Alert Triage & Case Creation.

From https://www.youtube.com/watch?v=uXS6OKGZ1ko&list=PLljFlTO9rB155aYBjHw2InKkSMLuhWpxH&index=6:
> In this session, we cover our second of three common workflows in Security Onion - Ad Hoc Hunting.

From https://www.youtube.com/watch?v=IS2SOlDedPc&list=PLljFlTO9rB155aYBjHw2InKkSMLuhWpxH&index=7:
> In this session, we cover the third and final common workflow in Security Onion - Detection Engineering.
